cita-network/src/connection.rs

```diff
@@ -427,7 +429,7 @@ pub fn manage_connect(config_path: &str, rx: Receiver<DebouncedEvent>, task_send
 
 fn config_peer(config: &[PeerConfig]) -> Vec<(u32, SocketAddr, String)> {
```

Comment:
Use `init_config_peer` instead of `config_peer`? Then why change this function?

Reply:
The update configuration also depends on "config_peer".